packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```diff
@@ -14,12 +14,12 @@
   "pci_database_add_step5_title": "Configurez vos options",
   "pci_database_add_name": "Nommez votre service de base de données",
   "pci_database_add_network_mode_label": "Type de réseau",
-  "pci_database_add_network_mode_public": "Public",
-  "pci_database_add_network_mode_private": "Privé",
+  "pci_database_add_network_mode_public": "Réseau public (Internet) ",
+  "pci_database_add_network_mode_private": "Réseau privé (vRack)",
   "pci_database_add_network_mode_public_incompatible": "L'offre sélectionnée n'est pas compatible avec les réseaux publics.",
   "pci_database_add_network_mode_private_incompatible": "L'offre sélectionnée n'est pas compatible avec les réseaux privés.",
   "pci_database_add_privateNetwork_label": "Réseau privé attaché",
-  "pci_database_add_privateNetwork_description": "Par défaut, les bases de données possèdent un réseau public. Vous pouvez également ajouter un réseau privé, mais vous perdrez la configuration que vous venez d’établir.",
+  "pci_database_add_privateNetwork_description": "Seuls les services (serveurs dédiés, Vms, clusters K8s, ...) connectés a ce <b>réseau privé (vRack)</b> pourront communiquer avec votre base de données. ",
```

Comment:
FYI there is the ellipsis character in utf-8 which looks better : …

Reply:
I did not know about this character, thanks ! I changed it here: https://github.com/ovh/manager/pull/7460/commits/d4d4bed42d64929ae00b322f1c38f3b297c4bdb0